ruuvigw2mqtt/ruuvi/index.js

```diff
@@ -0,0 +1,12 @@
+const { parseData } = require("ruuvitag-parser");
+
+const decoder = (data) => {
+  try {
+    return data.indexOf("FF9904") !== -1 ? parseData(data) : null;
```

Comment:
I wasn't sure should I include this in the `ruuvitag-parser` module or not, so I decided to leave it here instead..